sale_blanket_order_margin/tests/test_sale_blanket_order_margin.py

```diff
@@ -0,0 +1,142 @@
+from datetime import date, timedelta
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestSaleBlanketOrdersMargin(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.blanket_order_obj = self.env["sale.blanket.order"]
```

Comment:
never use setUp unless you really need it, go for setUpClass pls :pray: 
And disable tracking, tnx!